spaceship.zsh

```diff
@@ -587,6 +596,42 @@ spaceship_ruby() {
     "$SPACESHIP_RUBY_SUFFIX"
 }
 
+# ELIXIR
+# Show current version of Elixir
+spaceship_elixir() {
+  [[ $SPACESHIP_ELIXIR_SHOW == false ]] && return
+
+  # Show versions only for Elixir-specific folders
+  [[ -f mix.exs || -n *.ex(#qN) || -n *.exs(#qN) ]] || return
+
+  local elixir_version
+
+  if _exists kiex; then
+    elixir_version="${ELIXIR_VERSION}"
+  elif _exists exenv; then
+    elixir_version=$(exenv version-name)
+  fi
+
+  if [[ $elixir_version == "system" ]] || [[ $elixir_version == "" ]]; then
```

Comment:
don't show elixir version for `system` like other sections do. Drop the `[ $elixir_version == "system" ]]` condition